apps/web/src/client/components/files-list/files-list.module.js

```diff
@@ -1,42 +1,17 @@
 import { buildErrorBanner } from './_html.js';
 
 const initFilesListModule = () => {
-	/** @type {NodeListOf<HTMLInputElement>} */
-	const fileCheckBoxes = document.querySelectorAll('input[name="selectedFilesIds[]"]');
-	/** @type {HTMLElement | null} */
-	const selectAllCheckBox = document.querySelector('input[name="selectAll"]');
```

Comment:
I refactored the client-side code for the selectAll checkbox. Now it's in a dedicated reusable module